src/burnchains/mod.rs

```diff
@@ -495,6 +497,7 @@ impl fmt::Display for Error {
             Error::TrySyncAgain => write!(f, "Try synchronizing again"),
             Error::UnknownBlock(block) => write!(f, "Unknown burnchain block {}", block),
             Error::CoordinatorClosed => write!(f, "ChainsCoordinator channel hung up"),
+            Error::SQLiteError(e) => write!(f, "SQLite error: {}", e),
```

Comment:
Minor, but using inline variables looks better:

```suggestion
            Error::SQLiteError(e) => write!(f, "SQLite error: {e}"),
```

Reply:
Ah, yeah. I should've learned this by now from your other PRs 😅.